community/kernel/src/main/java/org/neo4j/kernel/api/index/ArrayEncoder.java

```diff
@@ -19,31 +19,40 @@
  */
 package org.neo4j.kernel.api.index;
 
+import sun.misc.BASE64Encoder;
+
 import java.io.IOException;
 import java.io.OutputStream;
 import java.lang.reflect.Array;
 
-import sun.misc.BASE64Encoder;
-
 import org.neo4j.kernel.impl.util.Charsets;
 
 public class ArrayEncoder
 {
-    private static final byte[] LINE_ENDING = System.lineSeparator().getBytes();
     private static final BASE64Encoder base64Encoder = new BASE64Encoder()
     {
         @Override
         protected void encodeBufferPrefix( OutputStream out ) throws IOException
         {
-            // don't initialize the non-thread-safe state
+            // do nothing
```

Comment:
I'd like to keep the previous pattern of comments in these methods. Describe *why* the method is overridden, i.e. what is the behaviour we expect the base class to perform here that we are suppressing by overriding.